docs/source/examples/interactive-development.rst

```diff
@@ -0,0 +1,170 @@
+Start a Development Cluster
+===========================
+
+SkyPilot makes the interactive development very easy on clouds. It helps you:
+
+#. Launch: a cluster with GPU or any specific requirement with a single command.
+#. Connect: Quickly connect to the cluster with the cluster name.
+#. Autostop: Automatically stop the cluster after a period of time for cost savings.
+
+Launch
+------
+
+To launch a cluster with a cheap GPU for developemnt:
+
+.. code-block:: bash
+
+  # Launch a cluster with 1 NVIDIA GPU and syncing the local working directory to the
+  # cluster.
```

Comment:
```suggestion
  # Launch a cluster with 1 NVIDIA GPU and sync the local working directory to the
  # cluster. This can be launched as a pod in your Kubernetes cluster or a cloud VM.
```